src/Umbraco.Core/DependencyInjection/UmbracoBuilder.cs

```diff
@@ -114,9 +114,8 @@ private void AddCoreServices()
             // Adds no-op registrations as many core services require these dependencies but these
             // dependencies cannot be fulfilled in the Core project
             Services.AddUnique<IMarchal, NoopMarchal>();
-            Services.AddUnique<IProfiler, NoopProfiler>();
+            //Services.AddUnique<IProfiler, NoopProfiler>();
```

Comment:
can remove commented out code. do we still need NoopProfiler? it's being checked for but doesn't seem like we use/need it?

Reply:
We still new up `NoopProfiler` in `Umbraco.Web.Common.DependencyInjection.UmbracoBuilderExtensions.GetWebProfiler`. So it is still needed